style.css

```diff
@@ -0,0 +1,113 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 92px;
+  background: #3c3a39;
+  font-size: 18px;
+  color: #fff;
+  text-decoration: none;
+}
+
+h1 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  padding-top: 120px;
+  padding-right: 24px;
+  padding-left: 24px;
+  margin-bottom: 6px;
+}
+
+h2 {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  color: #fff5e1;
+  padding: 1px 24px;
+}
+
+p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  margin: 12px 0.1px;
+  padding: 1px 24px;
+}
+
+.headline {
+  height: 590px;
+  background-color: #1c1a19;
+  background-image: url('./media/Illustration-header-mobile.svg');
```

Comment:
[OPTIONAL] 
- [ ] Kindly have a look at the URL to the background image. You saved it in lowercase but referenced it here in uppercase. While that might work on some systems, others are case-sensitive and that prevents it from displaying for example on my system. 

![image](https://user-images.githubusercontent.com/29071984/176758796-6bbeaca0-c9b6-492b-a949-21445e8fc330.png)

Reply:
@cwaku Thank you for catching that! The previous reviewer mentioned it but I only renamed the file and forgot to change it in the CSS